.docker/dev/compose.yml.example

```diff
@@ -3,7 +3,7 @@ services:
   postgres:
     env_file:
       - .env
-    image: postgres:16
+    image: pgvector/pgvector:pg16
```

Comment:
Users who have already deployed pg17 may be unable to upgrade. A document is needed to tell users how to handle this situation, for example, users already using pg 17 should use pgvector:17, and users already using pg 16 should use pgvector:16.